lib/compat/wordpress-5.9/class-wp-rest-global-styles-controller.php

```diff
@@ -0,0 +1,586 @@
+<?php
+/**
+ * REST API: Gutenberg_REST_Global_Styles_Controller class
+ *
+ * @package    Gutenberg
+ * @subpackage REST_API
+ */
+
+if ( ! class_exists( 'WP_REST_Global_Styles_Controller' ) ) {
+	/**
+	 * Base Global Styles REST API Controller.
+	 */
+	class WP_REST_Global_Styles_Controller extends WP_REST_Controller {
```

Comment:
This is just the exact same class that is in Core, the previous version in the Gutenberg plugin was missing some changes that landed directly on Core https://github.com/WordPress/wordpress-develop/commit/9f0d8fdf0137180278bdb355ae26ac0ad73e3e89